apparmor.d/abstractions/bus/org.freedesktop.UPower

```diff
@@ -12,13 +12,9 @@
        member=GetAll
        peer=(name=:*, label=upowerd),
 
-  dbus send bus=system path=/org/freedesktop/UPower/devices/DisplayDevice
+  dbus send bus=system path=/org/freedesktop/UPower/devices/*
```

Comment:
That does not work:
- `peer=(name=org.freedesktop.UPower, label=upowerd),` and `peer=(name=:*, label=upowerd),` are two different thing, you cannot merge them
- Be verbose in this bus file: add a new entry only for `PropertiesChanged` (with both peer name and label)